### PR TITLE
numpydoc build for gaussian.py

### PR DESCRIPTION
**Pull request by stevenhua0320**

### CODE REVIEW

**Comment by Ainamacar** [comment]
The new docstrings for `_jacobianraw` and `_valueraw` don't quite correctly describe the relationship between the concept of FWHM (which can be used to describe the width of any Gaussian) and the `maxwidth` member, which is the maximum allowed value of FWHM.

For example, if `maxwidth = 0.3`, the FWHM of the peak is constrained to being between 0 and 0.3, but it could have any value in the range.  Moreover, the internal parameterization doesn't use FWHM directly (no `fwhm` member exists).  Instead, it uses the "effective width" `pars[1]`, which can be any real number.

(It would have been nice if I could have used FWHM directly, but without a width constraint I found that all sorts of physically implausible peaks were extracted.)

What changes are necessary?  For example, the `_jacobianraw` docstring should be updated to say:
"This method calculates the partial derivatives of a Gaussian function
with respect to its parameters, considering a limiting width. The Gaussian's
width approaches its maximum FWHM (`maxwidth`) as the effective width parameter
(`pars[1]`) tends to infinity."

So take another stab at it!  Also, similar ideas apply to gaussianoverr.py, when you get to it.

Also, I know I'm only speaking up when I see something to fix, but I think you're doing a phenomenal job on these updates, @stevenhua0320 !

File: src/diffpy/srmise/peaks/gaussian.py [hunk 1]
```diff
@@ -154,9 +161,18 @@ def scale_at(self, pars, x, scale):
         SrMiseScalingError if the parameters cannot be scaled.
 
         Parameters
-        pars: (Array) Parameters corresponding to a single peak
-        x: (float) Position of the border
-        scale: (float > 0) Size of scaling at x."""
+        ----------
+        pars : array-like
+            Parameters corresponding to a single peak
```
Comment (sbillinge) [inline src/diffpy/srmise/peaks/gaussian.py:166]: These should always begin with "The" so The parameters corresponding..."

**Review by sbillinge — COMMENTED** [review]
Please see inline comment.  I think all the docstring need this small update.

**Comment by sbillinge** [comment]
thanks guys, great to see!

**Pull request merged**